doc/instanceVisualization.md

```diff
@@ -439,6 +440,19 @@ App:
 
 Check the function **URL** and **secret** in the front-end configuration file.
 
+#### Audience validation failed
+
+If you encounter the following error:
+
+> IDX10214: Audience validation failed.
+>
+> Audiences: [...]
+>
+> Did not match: validationParameters.ValidAudience: [...]
+
+then you may need to change the list of allowed audiences in the Authentication panel of your Function App (refer to section 2.2 “Configuration“ of the current page).
```

Comment:
Upper case on `then` ?

Reply:
I don't think this is necessary, because `then` is the start of the second half of the sentence that starts with `If you encounter the following error:`